salt/modules/aptpkg.py

```diff
@@ -164,7 +164,7 @@ def _call_apt(args, scope=True, **kwargs):
     '''
     cmd = []
     if scope and salt.utils.systemd.has_scope(__context__) and __salt__['config.get']('systemd.scope', True):
-        cmd.extend(['systemd-run', '--scope'])
+        cmd.extend(['systemd-run', '--scope', '--description "{0}"'.format(__name__)])
```

Comment:
This is making `apt` calls to produce the following issue on Ubuntu 1804:

```
    - systemd-run: unrecognized option '--description "salt.loaded.int.module.aptpkg"'
```

It seems the issue is caused due the "whitespace" introduced on the list of "args". This should be fixed like this in order to prevent the issue:

```suggestion
        cmd.extend(['systemd-run', '--scope', '--description', '"{0}"'.format(__name__)])
```